test/Fencer/Rules/Test.hs

```diff
@@ -4,7 +4,13 @@
 {-# LANGUAGE OverloadedLabels  #-}
 
 -- | Tests for "Fencer.Rules".
-module Fencer.Rules.Test (tests) where
+module Fencer.Rules.Test
+  ( tests
+  , writeAndLoadRules
+  -- example values
+  , domain1Text
```

Comment:
Let's not have different `*.Test` files depending on each other.

Let's also not have something like `domain1Text` in an export list, the name does not describe the semantics.

You can have a module called `RuleExamples` or something and reexport rules from there.

Reply:
I created a separate module `Fencer.Rules.Test.Examples` with these example values and renamed them to better suite what they represent.